src/util/cilfacade.ml

```diff
@@ -430,8 +430,12 @@ let stmt_fundecs: fundec StmtH.t Lazy.t =
     h
   )
 
+let pseudo_return_to_fun = StmtH.create 113
+
```

Comment:
I don't think it does because the `find_fundec` for nodes should only be used for nodes from the CFG of the current file. But I think there might be a problem adding the pseudo-return nodes to this single hash table during CFG creation. Because if several different CFGs (old and current) are created for an analysis run (e.g. during CFG comparison) then both the old as well as the new pseudo-return nodes are added to this table. Since pseudo-return nodes have consistent ids for unchanged functions and unique ids for changed functions, this might still work, but one would not expect the old pseudo-return nodes in a map which otherwise only maps statements from the current CFG.